src/elements/common/content-answers/__tests__/ContentAnswers.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Notification } from '@box/blueprint-web';
+import userEvent from '@testing-library/user-event';
+import { fireEvent, render, screen } from '../../../../test-utils/testing-library';
+
+import ContentAnswers from '../ContentAnswers';
+import { mockApi, mockFile } from '../__mocks__/mocks';
+// @ts-ignore: no ts definition
+import APIContext from '../../api-context';
+
+describe('common/content-answers/ContentAnswers', () => {
+    const renderComponent = (props?: {}) =>
+        render(
+            <Notification.Provider>
+                <Notification.Viewport />
+                <APIContext.Provider value={mockApi}>
+                    <ContentAnswers file={mockFile} {...props} />
+                </APIContext.Provider>
+            </Notification.Provider>,
+        );
+
+    test('should render the content answers', async () => {
+        renderComponent();
+
+        const button = screen.getByRole('button', { name: 'Box AI' });
+        expect(button).toBeInTheDocument();
+        await userEvent.click(button);
+        const modal = screen.getByTestId('content-answers-modal');
+        expect(modal).toBeInTheDocument();
+    });
+
+    test('should highlight open button when there is a question asked and the modal has closed', async () => {
+        renderComponent();
+
+        const button = screen.getByRole('button', { name: 'Box AI' });
+        await userEvent.click(button);
+
+        const modal = screen.getByTestId('content-answers-modal');
+        expect(modal).toBeInTheDocument();
+
+        const textArea = screen.getByRole('textbox', { name: 'Ask anything about this doc' });
+        fireEvent.change(textArea, { target: { value: prompt } });
```

Comment:
the `prompt` here seems not defined in this file. 

And could we try using `userEvent.type(textArea, 'question prompt')`. Same for ones used in ContentAnswersModal.test.tsx file. 
Example https://github.com/box/box-ui-elements/blob/master/src/elements/content-explorer/__tests__/RenameDialog.test.tsx#L59-L62